### PR TITLE
fix: make build pass

### PR DESCRIPTION
**Pull request by georgettica**
- change types of objects as requested
- change metav1.Convert_Map_string_To_string_To_v1_LabelSelector to correct format
- remove old repo links

### CODE REVIEW

File: PROJECT [hunk 1]
```diff
@@ -1,6 +1,6 @@
 domain: openshift.io
 layout: go.kubebuilder.io/v2
-repo: github.com/RiRa12621/openshift-route-monitor-operator
+repo: github.com/openshift-route-monitor-operator
```
Comment (NautiluX) [inline PROJECT:3]: isn't it `openshift/route-monitor-operator` ?

File: controllers/routemonitor_controller.go [hunk 2]
```diff
@@ -142,14 +142,19 @@ func (r *RouteMonitorReconciler) deploymentForBlackboxExporter(m *monitoringv1al
 
 // deploymentForServiceMonitor returns a ServiceMonitor
 func (r *RouteMonitorReconciler) deploymentForServiceMonitor(m *monitoringv1alpha1.RouteMonitor) *monitoringv1.ServiceMonitor {
-	var ls metav1.LabelSelector = metav1.Convert_Map_string_To_string_To_v1_LabelSelector(labelsForRouteMonitor(m.Name), &metav1.LabelSelector{}, nil)
+	var ls *metav1.LabelSelector
+	routeMonitorLabels := labelsForRouteMonitor(m.Name)
+	err := metav1.Convert_Map_string_To_string_To_v1_LabelSelector(&routeMonitorLabels, ls, nil)
+	if err != nil {
+		panic(1)
```
Comment (NautiluX) [inline controllers/routemonitor_controller.go:149]: yikes! Isn't there a more graceful way or at least inform why we panic?

**Review by NautiluX — CHANGES_REQUESTED** [review]

File: controllers/routemonitor_controller.go (hunk 2 above)
Comment (georgettica) [reply controllers/routemonitor_controller.go:149]: you are correct, I just wanted to make the build pass quickly, will fix XD

File: PROJECT (hunk 1 above)
Comment (georgettica) [reply PROJECT:3]: yeah, I did the fix like 5 times and I missed this one, will fix

File: go.sum [hunk 3]
```diff
@@ -73,6 +73,8 @@ github.com/PuerkitoBio/purell v1.1.0/go.mod h1:c11w/QuzBsJSee3cPx9rAFu61PvFxuPbt
 github.com/PuerkitoBio/purell v1.1.1/go.mod h1:c11w/QuzBsJSee3cPx9rAFu61PvFxuPbtSwDGJws/X0=
 github.com/PuerkitoBio/urlesc v0.0.0-20160726150825-5bd2802263f2/go.mod h1:uGdkoq3SwY9Y+13GIhn11/XLaGBb4BfwItxLd5jeuXE=
 github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578/go.mod h1:uGdkoq3SwY9Y+13GIhn11/XLaGBb4BfwItxLd5jeuXE=
+github.com/RiRa12621/openshift-route-monitor-operator v0.0.0-20200827095438-67fbb79f804a h1:8FFLSGAapF8JdDdSoiulCLGAwGc1X54yEHdWicJYqKA=
```
Comment (RiRa12621) [inline go.sum:76]: Are we sure that'll work in the future?

Comment (georgettica) [reply go.sum:76]: I will `go mod tidy` so they will be deleted

**Comment by georgettica** [comment]
squashed all commits, please lgtm if all is fine

**Review by RiRa12621 — APPROVED** [review]
/lgtm

**Comment by georgettica** [comment]
yay!

**Pull request merged**